content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,270 @@ you can view processes in other containers.
 
 * Learn how to [debug pods using ephemeral containers](/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container).
 
+### Kubernetes Ephemeral Container Security
```

Comment:
I would move this into [an existing] page within https://kubernetes.io/docs/concepts/security/ and then have a _link_ to that new section.